package.json

```diff
@@ -14,13 +14,14 @@
     "build": "tsc --build --verbose ./tsconfig.build.json",
     "lint": "eslint --fix src",
     "style": "prettier --write ./src/**/*.ts",
-    "test": "mocha -r ts-node/register 'test/**/*.test.ts' --exit",
+    "test": "mocha -r ts-node/register 'test/**/*.test.ts' --exit --timeout 25000",
```

Comment:
nit: it might be better to place this in a `.mocharc`

Reply:
good call, made this change